invenio_vocabularies/contrib/common/ror/datastreams.py

```diff
@@ -38,11 +44,28 @@ def read(self, item=None, *args, **kwargs):
         # See: https://github.com/inveniosoftware/rfcs/blob/master/rfcs/rdm-0071-signposting.md#provide-an-applicationlinksetjson-endpoint
         headers = {"Accept": "application/linkset+json"}
         api_url = "https://zenodo.org/api/records/6347574"
```

Comment:
1. `HEAD https://doi.org/10.5281/zenodo.6347574`
2. `GET $links.linkset.url -H "Accept: $links.linkset.type"`
3. Continue as is...